.rubocop_todo.yml

```diff
@@ -76,6 +76,30 @@ RSpec/ExampleWording:
 RSpec/MultipleExpectations:
   Max: 4
 
+# Offense count: 2
+# This cop supports unsafe autocorrection (--autocorrect-all).
+# Configuration parameters: ResponseMethods.
+# ResponseMethods: response, last_response
+RSpecRails/HaveHttpStatus:
+  Exclude:
+    - 'spec/requests/otp_requests_controller_spec.rb'
```

Comment:
Is there a reason to add these exclusions? Can we fix the warnings/errors instead? Same comment applies to the other exclusions added below

Reply:
it's auto generated with the comments. Better we can solve all the runocop issues in another PR. Also need to discuss enable/disable cops.

Reply:
Oh okay strange. Please ensure we have a ticket added in our backlog for the same and tag it here before we mark this discussion as resolved. Otherwise we might miss it out

Reply:
done. #PLM -28 and #PLM-29